src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -29,29 +29,26 @@ public MethodWithToken(MethodDesc method, ModuleToken token)
         }
     }
 
-    public class MethodContext
+    public struct MethodContext : IEquatable<MethodContext>
```

Comment:
`MethodContext` may be a bit misleading name for this. We seem to call it "generic context" in other places: https://github.com/dotnet/corert/blob/master/src/ILCompiler.Compiler/src/Compiler/GenericDictionaryLookup.cs#L123

Reply:
Good suggestion, I have renamed the class to "GenericContext" in my 3rd commit.